src/Microsoft.AspNet.Mvc.Razor/Compilation/CompilerCache.cs

```diff
@@ -194,18 +196,17 @@ public class CompilerCache : ICompilerCache
             };
         }
 
-        private CompilerCacheEntry PopulateCacheSetContext(ICacheSetContext cacheSetContext)
+        private CacheEntryOptions GetCacheEntryOptions(string key, CompilerCacheEntry compilerCacheEntry)
```

Comment:
I think all you need here is the relativePath. Could we simplify this method to accept the `RelativeFileInfo` and use that on L#202 and L#204?